stable/ghost/templates/deployment.yaml

```diff
@@ -17,7 +17,8 @@ spec:
     spec:
       initContainers:
       - name: volume-permissions
-        image: busybox
+        image: "{{ .Values.initContainerImage }}"
+        imagePullPolicy: {{ default "" .Values.imagePullPolicy | quote }}
```

Comment:
Even though this is not exactly as proposed, this still follows the same pattern as rest of the chart.   I think it's recommended to have ImageName , ImageTag and PullPolicy per container as its easy to configure in the future.